pages/1.11/release-notes/1.11.6/index.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.6
+title: Release Notes for 1.11.6
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.6
+---
+
+DC/OS 1.11.6 was released on September 25, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.6/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+
+DC/OS 1.11.6 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/19d17cec3e797758e76c081efb68867d440ed4d3/CHANGELOG).
+- Marathon 1.6.549 [change log](https://github.com/mesosphere/marathon/tree/aabf74302).
+- Metronome 0.4.34 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.4).
```

Comment:
Metronome Version is 0.4.4 ( the URL is correct). but the version information mentioned is incorrect.